src/components/folder-dropzone/folder-dropzone.js

```diff
@@ -156,7 +106,10 @@ export default class FolderDropzone extends React.Component {
     const isFileDefined = expectToBeDefined(event.dataTransfer.files[0]);
 
     if (!isFileDefined) {
-      notifyError(loadingErrorMessage, loadingErrorTitle);
+      notifyError(
+        this.props.t("folderDropzone.loadingErrorMessage"),
```

Comment:
destructurer les props pour éviter d'écrire (this.props) partout. Et définir le namespace dans le hoc.